config/electron.config.js

```diff
@@ -1,23 +1,25 @@
 const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
+const {version} = require('../package.json');
 
 module.exports = {
     appId: 'com.expensifyreactnative.chat',
     productName: 'New Expensify',
     extraMetadata: {
-        main: './desktop/main.js',
+        version,
         electronEnvironment: process.env.SHOULD_DEPLOY_PRODUCTION ? ENVIRONMENT.PRODUCTION : ENVIRONMENT.STAGING,
     },
     mac: {
         category: 'public.app-category.finance',
+        target: [
+            {target: 'dmg', arch: ['x64', 'arm64', 'universal']},
+        ],
```

Comment:
We can drop the `universal` target if we're not going to use it.
The idea is to give x64 to "old" mac users and `arm64` for M1s 
Universal works for both but is 2x the size (This was briefly touched on the issue as well: https://github.com/Expensify/App/issues/6927#issuecomment-1024510582)